lib/js/jslib.js

```diff
@@ -806,6 +806,14 @@ const _$Links = (function() {
       return true;
     }
 
+    if (_$Types.isDateTime(l) && _$Types.isDateTime(r)) {
```

Comment:
I think this code block needs to be moved up before the generic object comparison code. Otherwise I don't think this code will get executed properly.

Reply:
Whoops, good catch.

Reply:
(Funnily enough I noticed and fixed this for a few other cases -- this one slipped through the net! I'm grateful for you spotting it.)